tools/src/dbgen/Main.hs

```diff
@@ -142,10 +141,6 @@ newWalletState recreate walletPath =
     -- to rebuild the DB, but rather append stuff into it.
     liftIO $ openState (not recreate) walletPath
 
-instance HasLoggerName IO where
```

Comment:
this can savely be removed, as with the new logging all 'HasLoggerName' will disappear.
a LoggerName was used as a context name in logging. We will have a new mechanism which will cover this.

Reply:
Great, thanks @CodiePP.